### ISSUE TITLE
EditText: Keep text horizontally aligned

### ISSUE THREAD

**Issue by stefan-niedermann** [yydcdut/RxMarkdown#9]
This would be a feature that could enhance the readability of the text in edit mode:
all "content" of the user should be horizontally aligned and leading markdowns should get a negative offset.
Example: This is how [Nextcloud Notes (server app)](http://github.com/nextcloud/notes/) does it:

![bildschirmfoto am 2016-12-10 um 09 48 40-fullpage](https://cloud.githubusercontent.com/assets/4741199/21072304/dc2b6bbe-bebd-11e6-8fff-0aef3acff8ca.png)

**Comment by korelstar** [comment]
Yes, that would be nice. However, on mobile devices we typically have very few space in the horizontal direction. But such an alignment will consume some space which is lost for the actual text. So I'm not sure, if this is really desirable.

**Comment by stefan-niedermann** [comment]
Partally right: The only thing that could cause problems are H2 - H6 imho. Everything else does not need much space (lists, h1, checkboxes, quotes, ...) And H2 - H6 could behave like they do now - they will need more space. But all the other "markdowns" would be aligned much better with very view whitespace at the left side. (Hope you understand what i am trying to tell you 😆 )

**Comment by korelstar** [comment]
Yes, I think I do understand. :smiley:
But there are more patterns which do not fit into this schema: e.g. nested lists and nested quotes. Maybe, it would be sufficient, if the additional margins (e.g. for headings and lists) are removed, the following lines are indented accordingly and the markdown symbols are shown in gray.
